README.md

```diff
@@ -48,7 +48,7 @@ For Maven based projects, use the following to configure the CloudEvents Java SD
 <dependency>
     <groupId>io.cloudevents</groupId>
     <artifactId>cloudevents-core</artifactId>
-    <version>2.0.0-SNAPSHOT</version>
+    <version>2.0.0-milestone1</version>
```

Comment:
Consider using `2.0.0-M1`, it is a very popular scheme for milestones (and `-RC1` for releases)